x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/missing_transaction_warning.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export function MissingTransactionWarning() {
+  return (
+    <EuiToolTip
+      position="left"
+      content={i18n.translate(
+        'xpack.apm.transactionDetails.agentMissingTransactionMessage',
+        {
+          defaultMessage:
+            'This trace contains spans from missing transactions. As a result these spans are not displayed in the timeline.',
```

Comment:
Thank you, I remember I took the message from the issue description, and now that you mentioned that it makes sense. One question regarding the `{{count}}` - we want to show the missing items in the waterfall based on the items returned and the items shown in the waterfall, right? I guess here we just calculate it, I don't see the missing items count existing somewhere (similar to the check we have here but instead of returning a boolean it should return the missing items count)
<img width="1719" alt="image" src="https://github.com/elastic/kibana/assets/14139027/051f7c24-d3f5-42d9-932f-9b99ac33da78">

Reply:
I added a [draft PR](https://github.com/elastic/kibana/pull/173196) for those changes so we can better discuss there

Reply:
>  One question regarding the {{count}} - we want to show the missing items in the waterfall based on the items returned and the items shown in the waterfall, right? I guess here we just calculate it, I don't see the missing items count existing somewhere (similar to the check we have here but instead of returning a boolean it should return the missing items count)

Yes, I was thinking it would be beneficial to at least show the number of (orphan) items that cannot be placed because a parent is missing.